views/js/controller/export/index.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; under version 2
+ * of the License (non-upgradable).
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2017 (original work) Open Assessment Technologies SA;
+ *
+ */
+define(['jquery'], function($){
+    'use strict';
+
+    return {
+        start : function start(){
+            var $container = $('#exportChooser');
+
+            /**
+             * toggle the state of the submitter (active/disable) according to number of checked elements
+             * @param $container
+             */
+            var updateSubmitter = function updateSubmitter(){
+                var $formSubmitter = $container.find('.form-submitter');
+                if($container.find('.form-group :checkbox:checked').length > 0){
+                    $formSubmitter.removeClass('disabled');
+                }else{
+                    $formSubmitter.addClass('disabled');
+                }
+            };
+
+            updateSubmitter();
+            $container.on('change', ':checkbox', updateSubmitter);
+        }
+    };
+});
```

Comment:
so I made changes to https://github.com/oat-sa/tao-core/pull/1314 so this PR is no longer required